crates/nu-command/src/conversions/fill.rs

```diff
@@ -47,6 +47,10 @@ impl Command for Fill {
                 (Type::Float, Type::String),
                 (Type::String, Type::String),
                 (Type::Filesize, Type::String),
+                (Type::List(Box::new(Type::Int)), Type::List(Box::new(Type::String))),
+                (Type::List(Box::new(Type::Float)), Type::List(Box::new(Type::String))),
+                (Type::List(Box::new(Type::String)), Type::List(Box::new(Type::String))),
+                (Type::List(Box::new(Type::Filesize)), Type::List(Box::new(Type::String))),
```

Comment:
Need to clarify the exact behavior when given a list, maybe also the need to relax for heterogenous list

Reply:
from one of the examples of `fill`, maybe
```nushell
[1 2 3] | fill --alignment right --character '0' --width 5
```
could give
```
╭───┬───────╮
│ 0 │ 00001 │
│ 1 │ 00002 │
│ 2 │ 00003 │
╰───┴───────╯
```

Reply:
Yeah, i checked with `0.82.0` that it was previously supported and reenabled it.